app/content-panel/blogs/[id]/BlogDetailsPage.module.scss

```diff
@@ -50,6 +50,7 @@
     font-weight: bold;
     font-size: xx-large;
     width: 100%;
+    color: #a5a4a4; 
```

Comment:
use colors that are defined in _colors.scss
same for typography, we have all the styles defined in _typography.scss